docs/installation.md

```diff
@@ -0,0 +1,88 @@
+# Installation
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+## Prerequisites
+
+This project requires the following accounts and keys:
+
+* [Algolia](https://www.algolia.com)\* - App ID; API key (all operations); API key (search only)
+* [Auth0](https://auth0.com) - Domain; Client ID
+* [Google Cloud](https://cloud.google.com)\* - Application ID
+* [Graphcool](https://www.graph.cool) - Target; Endpoint URL [local instance available]
+* [Mailgun](https://www.mailgun.com)\* - Domain; API key
+
+\* _optional, but recommended_
+
+## Installing
+
+First, clone the repository
+
+```bash
+# Path: ./
+git clone https://github.com/Zenika/FAQ
+```
+
+Then, install the dependencies of the project
+
+```bash
+# Path: ./FAQ/
+npm install
+```
+
+## Configuration
+
+Before running the project, you will need to configure your environment variables
```

Comment:
I think this needs more info on how to do it, specifically on what values to put when developping with a local graphcool instance.